config/jobs/kubernetes-sigs/cluster-api-provider-cloudstack/capi-provider-cloudstack-presumbit.yaml

```
@@ -0,0 +1,125 @@
+presets:
+- labels:
+    preset-image-build: "true"
+  env:
+  - name: AWS_SDK_LOAD_CONFIG
+    value: "true"
+  - name: GOPROXY
+  volumeMounts:
+  - name: docker-registry-config
+    readOnly: false
+    mountPath: /root/.docker
+  - name: run-buildkit
+    readOnly: false
+    mountPath: /run/buildkit
+  - name: entrypoint
+    readOnly: false
+    mountPath: /script
+  - name: status
+    readOnly: false
+    mountPath: /status
+  volumes:
+  # The files in this repo update the respective configmaps
+  # via the config-updater plugin in plugins.yaml for hook
+  - name: docker-registry-config
+    configMap:
+      name: build-setup
+      items:
+      - key: docker-ecr-config.json
+        path: config.json
+  - name: entrypoint
+    configMap:
+      name: buildkitd-entrypoint
+      items:
+      - key: buildkitd-entrypoint.sh
+        path: entrypoint.sh
+  - name: run-buildkit
+    emptyDir: {}
+  - name: status
+    emptyDir: {}
+- env:
+  - name: AWS_ROLE_SESSION_NAME
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+
+
+presubmits:
+  kubernetes-sigs/cluster-api-provider-cloudstack:
+    - name: "capi-provider-cloudstack-presubmit-build"
+      labels:
+        preset-service-account: "true"
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+      always_run: false
+      run_if_changed: ".*"
+      max_concurrency: 10
+      decorate: true
+      decoration_config:
+        timeout: 1h
+      skip_report: false
+      clone_uri: "git@github.com:kubernetes-sigs/cluster-api-provider-cloudstack.git"
```

Is this required? if I'm not wrong 
```suggestion
presubmits:
  kubernetes-sigs/cluster-api-provider-cloudstack:
```
Is enough

---

Tested and verified, pushed an update to omit the `clone_uri` line.